pkg/config/v1/common.go

```diff
@@ -115,3 +116,12 @@ type HTTPPluginOptions struct {
 type HeaderOperations struct {
 	Set map[string]string `json:"set,omitempty"`
 }
+
+type AuthRetryConfig struct {
+	// MaxRetries specifies the maximum number of retries to authenticate
+	// with frps. By default, this value is 0.
```

Comment:
However, this is an incompatible modification, and if someone hasn't set this value, it will use the default value of 0, which will result in a failure to retrieve the OIDC token.

Reply:
I think you should not add a break-change code.